.gitignore

```diff
@@ -10,6 +10,7 @@
 
 # Ignore the default SQLite database.
 /db/*.sqlite3
+/doc/*
```

Comment:
I installed my ri docs necessitating ignoring this. Also the readme contained within this folder was superfluous.